browser-extension/src/addTournesolRecommendations.js

```diff
@@ -1,5 +1,7 @@
 // Youtube doesnt completely load a video page, so content script doesn't lauch correctly without these events
 
+import { millifyViews } from './utils';
```

Comment:
Hi, the new import syntax worked for me initially but did not afterward. So I have defined the millifyViews and the (still developing) timeAgo functions in `addTournesolRecommendations.js` itself rather than in `./utils`.

I am currently coding the time-ago based on published date, and classifying it based on today, yesterday, month, year, etc. and not more precise than that because the metadata didn't have `hours`, just the `date`.
This will be done soon.

Here's what the current output looks like: 
![Tournesol Demo 1](https://user-images.githubusercontent.com/20269286/218294475-f11afc8f-7de9-4f10-87a5-81d71c6b1108.JPG)